package.json

```diff
@@ -9,9 +9,10 @@
     "custom-login-server": "npm start --prefix custom-login/",
     "test:custom-login": "protractor okta-oidc-tck/e2e-tests/custom-login/conf.js",
     "resource-server": "node samples-nodejs-express-4/resource-server/server.js",
-    "test:e2e": "export TEST_TYPE=implicit && npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
-    "get-oidc-tck": "[ ! -d okta-oidc-tck/ ] && git clone https://github.com/okta/okta-oidc-tck.git",
-    "pretest": "webdriver-manager update --gecko false"
+    "test:e2e": "cross-env TEST_TYPE=implicit && npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
+    "get-oidc-tck": "[ ! -d okta-oidc-tck/ ] && git clone https://github.com/okta/okta-oidc-tck.git || echo \"TCK already cloned\"",
+    "pretest": "webdriver-manager update --gecko false",
+    "postinstall": "(cd custom-login && npm install) && (cd ../okta-hosted-login || cd okta-hosted-login && npm install)"
```

Comment:
For some weird reason, `"postinstall": "(cd custom-login && npm install) && (cd ../okta-hosted-login && npm install)"` works on windows but doesn't work on unix.
Hence I had to add the hack `(cd ../okta-hosted-login || cd okta-hosted-login && npm install)`

Reply:
Can this be simplified by using the prefix option when calling npm?

Reply:
If we use the prefix option, it fails on windows as it goes into a kind of an infinite loop. Since `postinstall` contains `npm install --prefix`, after it completes, it calls `postinstall` again which fails after sometime. It's not an issue on unix though.

Reply:
We *should* also try and split up the logic here, as it is a bit confusing:

```
"postinstall": "npm run install-custom-login && npm run install-okta-hosted-login",
"install-custom-login": "cd custom-login && npm install",
"install-okta-hosted-login": "cd okta-hosted-login && npm install",
...
```